csv.inc.php

```diff
@@ -45,7 +44,6 @@ protected function csvToArray($csv = 'stock.csv')// Access and extract CSV
                     $this->fieldTitles = $row; // Store titles in the code to be placed when required
                     $this->num= count($this->fieldTitles);
```

Comment:
Lines like this should have consistent formatting, like this:
$this->num = count($this->fieldTitles);